src/apps/loan-list/materials/utils/material-fetch-hoc.tsx

```diff
@@ -8,12 +8,14 @@ import { Product } from "../../../../core/publizon/model";
 import { BasicDetailsType } from "../../../../core/utils/types/basic-details-type";
 import { mapManifestationToBasicDetailsType } from "../../../../core/utils/helpers/list-mapper";
 import { ListType } from "../../../../core/utils/types/list-type";
+import { ILLBibliographicRecord } from "../../../../core/fbs/model";
```

Comment:
We should distinguish between our domain language and the domain language of the services.
I do not understand ILLBibliographicRecord. I discussed it with Claus and we agreed that it is probably and old way of defining a material record in FBS.
I think we should take ownership of that kind of entity and define it in our pool of entities: `src/core/utils/types/entities.ts`. We could call the type something like:  `LegacyFbsMaterial`and use that as a prop name as well.